playwright/utils/mockManagerProcedures.ts

```diff
@@ -19,7 +19,7 @@ type MockManagerProceduresArgs = {
     /**
      * Function that takes the existing data and returns the data to return.
      */
-    data?: (data: Record<string, unknown>) => Record<string, unknown>;
+    data?: (data: unknown, args: unknown[]) => unknown;
```

Comment:
True that it can be better to have unknown instead of all possible use cases. Anyway, we could improve that with the typing I failed to do depending on the manager procedure.

Reply:
Yes, we can definitely improve on even the `unknown` changes by using real typing.